docs/developer/LAPIS.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
+'https://plantuml.com/component-diagram
+
+package "External Databases" {
+    database "GISAID/\nOpen" as externalDB
+'    database Open
+}
+
+package LAPIS {
+    [In Memory DB]
+    [API]
+  [Update Pipeline]
+}
+
+package "Cov-Spectrum-Server" {
+    [Server API]
+    database "???"
```

Comment:
The server provides COVID-19 case data from Our World in Data (OWID) and Pango lineage alias data. They are imported by these scripts:

- https://github.com/GenSpectrum/cov-spectrum-server/tree/develop/ext_models/ingest-owidCases
- https://github.com/GenSpectrum/cov-spectrum-server/tree/develop/ext_models/ingest-pangoLineageAliases

They are directly running on GitHub:

- https://github.com/GenSpectrum/cov-spectrum-server/actions/workflows/run_ingest-owidCases.yml
- https://github.com/GenSpectrum/cov-spectrum-server/actions/workflows/run_ingest-pangoLineageAliases.yml